src/dmd/cli.d

```diff
@@ -336,10 +336,8 @@ dmd -cov -unittest myprog.d
         Option("g",
             "add symbolic debug info",
             `$(WINDOWS
-                Add CodeView symbolic debug info with
-                $(LINK2 $(ROOT_DIR)spec/abi.html#codeview, D extensions)
```

Comment:
Sorry, why remove the link to https://dlang.org/spec/abi.html#codeview ?

Reply:
I removed the "extension" part as that only applies to OMF and no debugger really supports them (mago can interpret them, but you normally run the exe through cv2pdb that translates them back to something a C++ debugger can deal with), so it's not really relevant anymore.